### PR TITLE
Display Request Id on 'Registered Request' page. Issue 710

### LINKED ISSUE #710: Add request number in the /request page table  as first column
### Subject of the issue
Show the request number in the table in https://keralarescue.in/requests page . Currently there is no way to mark the requests as closed by the data collection and verification teams . Currently what we are doing is using call the person using the contact in one ticket , search with the mobile number in rescue site to get all duplicates and note down all the duplicates tickets . If you guys are developing the functionality for marking as closed , You also might need the request number to update the historical data . 

### Your environment
* version of your python
* which browser and its version
* OS Type (iOS/Android/Linux/Windows)

### Steps to reproduce
go to https://keralarescue.in/requests  page 

### Expected behaviour
Tell us what and how exactly is it supposed to  happen
 the table should contain the request number 
### Actual behaviour
No request number shown currently

### PR DESCRIPTION
**Pull request by and**
### Issue Reference
This PR addresses the Issue : Fixes #710 

### Summarize
This PR adds 'Request Id' column to the existing table.

> Currently all columns have equal width, we can reduce the width for 'Request Id' column.

### CODE REVIEW

**Comment by shajith** [comment]
@and can you rebase/squash?

**Comment by tachyons** [comment]
Remove unrelated commits from the PR

File: templates/home.html [hunk 1]
```diff
@@ -9,77 +9,77 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
     <p style="margin-bottom:20px;">
         <i>An initiative by Govt. of Kerala, Kerala State IT Mission and IEEE Kerala Section<br>
         For effective collaboration and communications between authorities, volunteers and public </i><br/>
-        <strong>ദുരിതാശ്വാസ പ്രവര്‍ത്തനങ്ങളും രക്ഷാ ദൗത്യങ്ങളും ഏകോപിപ്പിക്കാന്‍</strong><br/>
+        <strong><span class="notranslate">ദുരിതാശ്വാസ പ്രവര്‍ത്തനങ്ങളും രക്ഷാ ദൗത്യങ്ങളും ഏകോപിപ്പിക്കാന്‍</span></strong><br/>
```
[the remark this answers is not in the document]
Comment (realslimshanky) [reply templates/home.html]: Please add a comment here as well explaining that `notranslate` is used to restrict google translate.

**Comment by realslimshanky** [comment]
@and please resolve conflicts as well.

**Comment by shajith** [comment]
Ended up fixing this in https://github.com/IEEEKeralaSection/rescuekerala/pull/739

**Pull request closed without merging**